osu.Game/Online/API/Requests/SearchBeatmapSetsRequest.cs

```diff
@@ -5,11 +5,21 @@
 using osu.Game.Overlays;
 using osu.Game.Overlays.BeatmapListing;
 using osu.Game.Rulesets;
+using Newtonsoft.Json;
 
 namespace osu.Game.Online.API.Requests
 {
     public class SearchBeatmapSetsRequest : APIRequest<SearchBeatmapSetsResponse>
     {
+        public class Cursor
+        {
+            [JsonProperty("approved_date")]
+            public string ApprovedDate;
+
+            [JsonProperty("_id")]
+            public string Id;
```

Comment:
As I said in the discussion of the original PR this is not enough. If you change the sort criteria the cursor will contain different fields.

Reply:
Will be fixed

Reply:
I think this can be consider fixed for now, as far as I can tell

Reply:
I'm not sure how you can assert that. I see no new pushed changes that would introduce the other possible cursor fields.

Reply:
I forgot to public push, sorry - should be visible now